internal/oci/container.go

```diff
@@ -58,6 +60,10 @@ type Container struct {
 	stdin              bool
 	stdinOnce          bool
 	created            bool
+	// the unix start time of the process, as found by psgo
+	// this is used to track whether the pid we have stored
+	// is the same as the same pid number on the host
+	psStartTime        string
```

Comment:
How about `pidStartTime` ?

Reply:
yeah that's better. I was at a loss for a variable name :)

Reply:
`processStartTime` is probably better than that.

Reply:
This will have to be persisted in state.

Reply:
I am thinking that we should probably separate the pid returned from runc state from the container start pid. runc state could change to zero once stopped. We can sepearately store the startTime, Pid as top level fields in crio state that never change once populated.